setuptools_rust/build.py

```diff
@@ -162,12 +167,19 @@ def build_extension(self, ext):
                 args.extend(
                     ["-C", "link-arg=-undefined", "-C", "link-arg=dynamic_lookup"]
                 )
+            # Tell musl targets not to statically link libc. See
+            # https://github.com/rust-lang/rust/issues/59302 for details.
+            if b'target_env="musl"' in rust_target_info:
+                rustflags += " -C target-feature=-crt-static"
 
         if not quiet:
             print(" ".join(args), file=sys.stderr)
```

Comment:
If it is correct to change the env vars to `args.extend` then might want to move this `print` after all args have been appended.